src/number_theory/legendre_symbol/jacobi_symbol.lean

```diff
@@ -0,0 +1,532 @@
+/-
+Copyright (c) 2022 Michael Stol. All rights reserved.
```

Comment:
```suggestion
Copyright (c) 2022 Michael Stoll. All rights reserved.
```
I guess?

Reply:
Correct. Making mistakes where it hurts most...  :smile: